### PR TITLE
Add networking with TLS routing

### LINKED ISSUE #13751: Networking section is incomplete
## Details

The ports section does not mention TLS routing:

https://goteleport.com/docs/setup/reference/networking/#ports

It should mention that with TLS routing, all proxy ports collapse into one, 443. This should be the default, and the other mode with many ports is a legacy one.

### PR DESCRIPTION
**Pull request by klizhentas**
Fixes #13751

### CODE REVIEW

**Comment by klizhentas** [comment]
@zmb3 @r0mant is the new section accurate from the perspective of Desktop access and K8s access? I've noticed two exra ports, but dont' recall if TLS routing routes properly without those up.

File: docs/pages/setup/reference/networking.mdx [hunk 1]
```diff
@@ -114,17 +114,36 @@ numbers for each service.
...
+### Ports with TLS routing
+
+TLS routing is enabled by default. In this mode, all connections to a Teleport service (e.g., the Teleport
+SSH Service or Kubernetes) are routed through the Proxy Service's public web address.
+
+Read more in our [TLS Routing](../../architecture/tls-routing.mdx) guide.
+
+| Port | Service | Description |
+| - | - | - |
+| 443 | Proxy | In TLS Routing mode, Proxy handles all protocols, including Web UI, HTTPS, Kubernetes SSH, all databases on a single port. |
```
Comment (wadells) [inline docs/pages/setup/reference/networking.mdx:126]:
```suggestion
| 443 | Proxy | In TLS Routing mode, the Proxy handles all protocols, including Web UI, HTTPS, Kubernetes, SSH, and all databases on a single port. |
```

**Review by r0mant — APPROVED** [review]

File: docs/pages/setup/reference/networking.mdx [hunk 2]
```diff
@@ -114,17 +114,36 @@ numbers for each service.
...
+TLS routing is enabled by default. In this mode, all connections to a Teleport service (e.g., the Teleport
+SSH Service or Kubernetes) are routed through the Proxy Service's public web address.
+
+Read more in our [TLS Routing](../../architecture/tls-routing.mdx) guide.
+
+| Port | Service | Description |
+| - | - | - |
+| 443 | Proxy | In TLS Routing mode, the Proxy handles all protocols, including Web UI, HTTPS, Kubernetes, SSH, and all databases on a single port. |
+| 3022 | Node | SSH port. This is Teleport's equivalent of port `#22` for SSH. Only used when Teleport Node is replacing SSH.|
+| 3025 | Auth | SSH port used by the Auth Service to serve its API to other Nodes in a cluster.|
```
[the remark this answers is not in the document]
Comment (klizhentas) [reply docs/pages/setup/reference/networking.mdx]: @strideynet good catch. I fixed SSH port which is a very old thing that is no longer true, now it's just TLS.

[the remark this answers is not in the document]
Comment (strideynet) [reply docs/pages/setup/reference/networking.mdx]: Perfect, that confirms my own research on it. I'd been searching high and low for somewhere the listener would accept a SSH connection 😅

**Review by strideynet — APPROVED** [review]
Looks good to me 👍

**Comment by zmb3** [comment]
Port 3028 for desktop access should be present in both lists as windows desktop service is always going to have a listen port when you're not using iot mode. 

TLS routing is a feature of the proxy service. Desktop access connections to the proxy already come in on the proxy https port because you have to use the web UI to start a Session.

**Pull request merged**